README.adoc

```diff
@@ -27,11 +29,30 @@ Updatecli reads a yaml or a go template configuration file, then works into thre
 
 == Why
 
-They are many projects out there, which can continuously update your files but they all have an opinionated way of doing it and they often want you to adopt a new platform. Building and distributing software is a difficult thing and good practices constantly evolve. Updatecli was built to work independently wherever you need, and combining whatever workflow you are using as you can see in the following section.
+They are many projects out there which can continuously update your files, but they all have an opinionated way of doing it and they often want you to adopt a new platform. Building and distributing softwares is a difficult task and good practices constantly evolve. Updatecli was built to work independently wherever you need, and combining whatever workflow you are using, as you can see in the following section.
+
+== Installation
+
+Updatecli is a simple Go binary, available for Linux, MacOS, Windows. It can be downloaded from the link:https://github.com/updatecli/updatecli/releases[release page], or installed via link:https://www.updatecli.io/docs/prologue/installation/[other methods].
```

Comment:
```suggestion
Updatecli is a Go binary available for Linux, MacOS and Windows from the link:https://github.com/updatecli/updatecli/releases[release page] or installed via link:https://www.updatecli.io/docs/prologue/installation/[other methods].
```

- Never use "just" or "simple": it's demotivating and is a poor english syntax :)
-Oxford comma (don't awake Tyler ;) )

Reply:
> Never use "just" or "simple": it's demotivating and is a poor english syntax :)
You are definitely right, but there are bad habits that are difficult to get rid of